### ISSUE TITLE
Incorrect function name in pop-up help for tp.file.move()

### ISSUE THREAD

**Issue by SteveDockar** [SilentVoid13/Templater#1136]
**Plugin information (please complete the following information):**

-   OS: [Windows 11]
-   Templater version: [1.16.0]
-   Obsidian version: [1.3.5]

**Describe the bug**
The pop-up help for _tp.file.move()_ is incorrectly labelled (and so only appears) as _tp.file.functions.move_.

**Expected behavior**
Entering `tp.file.m` should suggest the tp.file.move() function.

**Screenshots**
![image](https://github.com/SilentVoid13/Templater/assets/22272895/afe47bb3-33c6-401f-bda0-4a10ed7a2bcd)

**Additional context**
The pop-up help text correctly identifies the function's actual name, but is only triggered by the incorrect function name, and appears in the list of file module functions sorted under 'f' instead of 'm'. Hitting enter while the help text is displayed enters the incorrect tp.file.functions.move() string.

Checking `/docs/documentation.toml` shows the incorrect 'name' parameter in line 208.

```
[tp.file.functions.move]
name = "functions.move"
description = "Moves the file to the desired vault location."
definition = "tp.file.move(new_path: string, file_to_move?: TFile)"
```

**Comment by mariusgerome** [comment]
Hey. I tried both options (tp.file.functions.move and tp.file.move) and both don't work. But I'm also just starting out with this stuff and I might be … stupid.

I have a folder names "_new" directly in my obsidian root. And I want newly created daily, weekly, monthly and yearly notes (by the periodic notes plugin) to be moved to "_new" after they got created. So I tried this:
<% tp.file.functions.move(/_new/, TFile) %>
and this:
<% tp.file.move(/_new/, TFile) %>
and a bunch of variations (with "/" and without etc…) but nothing seems to work when I trigger the "Replace Templates in Active File" Command.

Btw I don't even understand what "TFile" stands for :D "This file" ? Maybe. Hm, perhaps I need to insert some other value?
– M

**Comment by SteveDockar** [comment]
> I have a folder named "_new" directly in my obsidian root. And I want newly created daily, weekly, monthly and yearly notes (by the periodic notes plugin) to be moved to "_new" after they got created. 

You don't need to get overly complex with this. You can include a path directly in the folder and format data for Periodic notes. I have a folder under root called Periodic-Notes and under that Daily-Notes, Weekly-Notes etc. In the Periodic Notes folder settings for Daily Notes I have `Periodic-Notes/Daily-Notes` then as I want sub-folders for Year/Month I then have the note _(file name)_ format set to `YYYY/MM/YYYY-MM-DD` so each note file is still named for the Year-Month-Date, but is under a Year and then Month folder in my /Periodic-Notes/Daily-Notes folder(s).

I realise this isn't exactly what you're aiming to do, but you can use /_new in the Periodic Notes "Note Folder" setting(s) and then also specify sub-folders or specific naming formats in the Format setting(s).

If you do want to use Templater, you can apply the move instruction at the top of a template applied to the note. If I tag a contact using `[[first last]]` and then decide to actually create a contact note/file for that link, clicking on it creates the note in the vault's root. I then apply a 'person' template which as well as all the stuff I want for the new person file inside it, has
```
<%*
tp.file.move("/People/" + tp.file.title)
_%>
```
at the top, which moves the file to a _new_ file in my /People folder with the same file name as the original.

I hope that helps. 
Dox.

**Comment by Zachatoo** [comment]
Thanks for the feedback on the docs being confusing. I updated the docs for `tp.file.move` to hopefully be a little more clear. Does it make more sense with the changes I made? Anything else you would change? There's also examples at the bottom of most pages that explain what each example is doing.

Link to tp.file.move docs: https://silentvoid13.github.io/Templater/internal-functions/internal-modules/file-module.html#tpfilemovenew_path-string-file_to_move-tfile
